pkg/apis/componentconfig/types.go

```diff
@@ -503,6 +503,8 @@ type KubeControllerManagerConfiguration struct {
 	// horizontalPodAutoscalerSyncPeriod is the period for syncing the number of
 	// pods in horizontal pod autoscaler.
 	HorizontalPodAutoscalerSyncPeriod unversioned.Duration `json:"horizontalPodAutoscalerSyncPeriod"`
+	// URL to an external heapster backend
+	Heapster string `json:"heapster"`
```

Comment:
I vote for `MetricsSource`. Also please extend the comment to what would happen if the address is not specified.